src/scss/pages/_learn-paths.scss

```diff
@@ -0,0 +1,34 @@
+@media (min-width: 865px) {
```

Comment:
It looks like this stylesheet replaces the old styles/pages/_path.scss. Please remove that stylesheet if not needed anymore.